.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Continuous Integration
+
+on:
+  release:
+    types: [created]
+
+permissions:
+  contents: read
+
+jobs:
+  phar:
+    name: Create a PHAR
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
+
+      - name: Install dependencies (lib)
+        run: composer install --prefer-dist --no-progress --no-dev --optimize-autoloader --classmap-authoritative
+
+      - name: Install dependencies (PHAR builder)
+        run: composer install --prefer-dist --no-progress --optimize-autoloader --classmap-authoritative
+        working-directory: tools/phar
+
+      - name: Compile PHAR
+        run: vendor/bin/box compile
+        working-directory: tools/phar
+
+      - name: Ensure PHAR is OK
+        run: build/castor.phar --version
+        working-directory: tools/phar
+
+      - name: Upload release binary
+        uses: actions/upload-release-asset@v1.0.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
Can't we re-use the build from the other workflow?

If we can, can you share this piece of YAML with the other workflow?

https://speakerdeck.com/lyrixx/github-actions-automatisation-de-vos-workflows?slide=13